style.css

```diff
@@ -149,31 +161,26 @@ footer {
 }
 
 
-.high_way {
-  margin: 0 8px;
-}
-
 .paragraph_header {
   font-weight: bold;
   margin-bottom: 24px;
+  margin-top: 32px;
 }
 
 .pros_photo {
-  width: 400px;
   height: 324px;
 }
 
-.paragraph_description {
+.constant_width {
   width: 400px;
-  line-height: 24px;
 }
 
-.header_container {
-  width: 400px;
+.paragraph_description {
+  line-height: 24px;
 }
 
 .text {
-color: rgba(0, 0, 0, 0.46);
+  color: #848383;
```

Comment:
rgba stands for: (redValue, greenVal, blueVal, alphaValue) - alpha is the opacity.
When you changed the color to hex value it's transparent 👍

Reply:
got it now :) it was written like this in figma :)

Reply:
I know, it's one of the things we should remember about